core/src/main/java/greencity/config/SwaggerConfig.java

```diff
@@ -56,7 +58,9 @@ public Docket swaggerSpringfoxDocket() {
                 .securitySchemes(Lists.newArrayList(apiKey()))
                 .useDefaultResponseMessages(false);
 
-        docket = docket.select().apis(RequestHandlerSelectors.withClassAnnotation(RestController.class)).build();
+        docket =
+            docket.select().apis(RequestHandlerSelectors.any())
```

Comment:
Can we specify Controller and RestController only instead of any?

Reply:
> Can we specify Controller and RestController only instead of any?

I did not find a solution for this